src/go/k8s/apis/redpanda/v1alpha1/cluster_types.go

```diff
@@ -33,6 +33,10 @@ type ClusterSpec struct {
 	Resources	corev1.ResourceRequirements	`json:"resources"`
 	// Configuration represent redpanda specific configuration
 	Configuration	RedpandaConfig	`json:"configuration,omitempty"`
+
+	// ExternalConnectivity add listener that can be reached outside
```

Comment:
adds.

maybe also add some more context as to how is it implemented?

Reply:
Thanks, Will add.